packages/nexus-sdk/src/Resource/index.ts

```diff
@@ -72,11 +72,20 @@ const Resource = (
       orgLabel: string,
       projectLabel: string,
       payload: ResourcePayload,
+      schemaId: string = '_',
+      resourceId?: string,
     ): Promise<Resource> =>
-      httpPost({
-        path: `${context.uri}/resources/${orgLabel}/${projectLabel}`,
-        body: JSON.stringify(payload),
-      }),
+      resourceId
+        ? httpPut({
+            path: `${context.uri}/resources/${orgLabel}/${projectLabel}/${schemaId}/${resourceId}`,
+            body: JSON.stringify(payload),
+          })
+        : httpPost({
+            path: `${context.uri}/resources/${orgLabel}/${projectLabel}/${
+              schemaId === '_' ? '' : schemaId
```

Comment:
If you make this `_` or `/schemaId`, I hen you don't have to add the trailing `/` I'm your unit test, which means you're sure you haven't changed the behaviour of the SDK for developers using this function in their code

Reply:
I have updated the code.